src/odemis/driver/test/zeiss_test.py

```diff
@@ -229,6 +231,108 @@ def test_move(self):
         dur = time.time() - tstart
         self.assertGreaterEqual(dur, 1.1, "1 mm move took only %g s" % dur)
 
+    def test_move_axes6_relative(self):
+        """
+        Test if it's possible to execute a relative move of the stage using 6 axes
+        Test if moving below 0° or over 360° using a full rotational axis applies correction of under- and overflow
```

Comment:
Not sure if the overflow condition is checked. If it means something like this, i.e. 360° + 0.1.